CodeEdit/Features/NavigatorArea/FindNavigator/FindNavigatorForm.swift

```diff
@@ -0,0 +1,270 @@
+                searchModel.append(secondMode)
+                if let thirdMode = secondMode.children.first, let selectedThirdMode = selectedMode.third {
+                    if secondMode.children.contains(selectedThirdMode) {
+                        searchModel.append(selectedThirdMode)
+                    } else {
+                        searchModel.append(thirdMode)
+                    }
+                }
+            }
+        }
+    }
+
+    private var chevron: some View {
+        Image(systemName: "chevron.compact.right")
+            .foregroundStyle(.tertiary)
+            .imageScale(.large)
+    }
+
+    var body: some View {
+        VStack {
+            HStack {
+                HStack(spacing: 0) {
+                    ForEach(0..<selectedMode.count, id: \.self) { index in
+                        if index > 0 {
+                            chevron
+                        }
+                        Menu {
+                            ForEach(getMenuList(index), id: \.title) { (searchMode: SearchModeModel) in
+                                Button(searchMode.title) {
+                                    onSelectMenuItem(index, searchMode: searchMode)
+                                }
+                            }
+                        } label: {
+                            Text(selectedMode[index].title)
+                                .foregroundColor(
+                                    selectedMode[index].needSelectionHighlight
+                                    ? Color.accentColor
+                                    : .primary
+                                )
+                                .font(.system(size: 11))
+                        }
+                        .searchModeMenu()
+                    }
+                    Spacer()
+                }
+                Spacer()
+                Text("Scoped")
+                    .controlSize(.small)
+                    .foregroundStyle(Color(nsColor: scoped ? .controlAccentColor : .controlTextColor))
+                    .onTapGesture {
+                        scoped.toggle()
+                    }
+            }
+            PaneTextField(
+                state.selectedMode[1].title,
+                text: $searchText,
+                axis: .vertical,
+                leadingAccessories: {
+                    Image(systemName: "magnifyingglass")
+                        .padding(.leading, 8)
+                        .foregroundStyle(.tertiary)
+                        .font(.system(size: 12))
+                        .frame(width: 16, height: 16)
+                },
+                trailingAccessories: {
+                    Divider()
+                    Button {
+                        caseSensitive.toggle()
+                    } label: {
+                        Image(systemName: "textformat")
+                            .foregroundStyle(caseSensitive ? Color(.controlAccentColor) : Color(.secondaryLabelColor))
+                    }
```

Comment:
Most of our tooltips seem to be broken (at least the ones on the icon buttons/toggles. I am not sure if this is my issue or something else because when hovering something that has a `.help`, I get this...
<img width="476" alt="Screenshot 2023-11-29 at 1 37 19 AM" src="https://github.com/CodeEditApp/CodeEdit/assets/806104/d36d8d88-012a-405d-b1ef-38d96f826a08">
<img width="741" alt="Screenshot 2023-11-29 at 1 38 16 AM" src="https://github.com/CodeEditApp/CodeEdit/assets/806104/4950d2bb-febd-4768-b6fe-6f2fa6e67303">
<img width="830" alt="Screenshot 2023-11-29 at 1 39 00 AM" src="https://github.com/CodeEditApp/CodeEdit/assets/806104/08e22c23-dd67-403c-93b0-b182f7836413">